src/main/resources/messages/LibertyBundles.properties

```diff
@@ -40,14 +40,16 @@ action.io.openliberty.tools.intellij.actions.RemoveLibertyProjectAction.descript
 
 # Messages for Liberty actions and pop-up dialog
 no.liberty.projects.detected=No Liberty Maven or Liberty Gradle projects detected in this workspace.
-liberty.dev.not.started.notification.content=Unable to {0}: Liberty dev mode may not have been started on {1}. \nUse the Liberty tool window to start Liberty dev mode.
+liberty.dev.not.started.notification.content=Unable to {0}. Liberty dev mode may not have been started on {1}. \nUse the Liberty tool window to start Liberty dev mode.
```

Comment:
There's always the risk of the translation team ending up making a mistake and not including the `\n`. I would prefer to escape it in another parameter, i.e., `{2}`, or even break the message into two parts. Since this is minor, I will approve it and let you decide if it's worth changing it or not.

Reply:
For escaping in another parameter, if I used `{2}` then I pass '\n' when I call `LocalizedMessageUtil.getMessage(...`?

Reply:
Updated to use a variable and pass `System.lineSeparator()`